microcosm_pubsub/result.py

```diff
@@ -41,9 +47,15 @@ def invoke(cls, func, message, **kwargs):
             else:
                 result = MessageHandlingResultType.SKIPPED
             message.ack()
+        except IgnoreMessage:
+            result = MessageHandlingResultType.IGNORED
+            message.ack()
         except SkipMessage:
             result = MessageHandlingResultType.SKIPPED
             message.ack()
+        except TTLExpired:
```

Comment:
This case would have been handled as `FAILED` previously (e.g. `nack` not `ack` and retry)